benchmarks/templates/benchmarks/leaderboard/leaderboard-table.html

```diff
@@ -0,0 +1,153 @@
+{% if models %}
+{% if has_user %}
+
+{% load static %}
+
+<form>
+{% endif %}
+
+<banner>
+  <div class="leaderboard-table-banner-icon gradient">
+      <img src="{% static "/benchmarks/img/leaderboard_header_icon.png" %}" alt="Brain-Score Logo">
+  </div>
+  <div class="leaderboard-table-banner box gradient"><h3>Leaderboard</h3></div>
+</banner>
+<div class="filters">
+  <select class="benchmark-sort">
+    <option value="" disabled selected>Sort by Benchmarks</option>
+    <option value="average">Sort by average</option>
+    {% for name in benchmark_names %}
+      <option value="{{name}}">{{ name }}</option>
+    {% endfor %}
+  </select>
+  <div class="legend" style="
+  ">
+    <label>Score Legend</label>
+    <span>No Alignment</span>
+    <span class="legend-right">Alignment at ceiling</span>
+  </div>
+  <div class="leaderboard-breadcrumb"></div>
+</div>
+<div class="leaderboard-table-container">
+  <table class="table">
```

Comment:
fyi for reviewing - this is pretty close to just straight up copy/pasted from the existing table. I cleaned it up a bit and removed some of the classes and stuff that i didn't need, but otherwise it should basically be 1:1